zero_bin/rpc/src/lib.rs

```diff
@@ -30,6 +30,14 @@ pub enum RpcType {
     Native,
 }
 
+#[derive(Clone, Debug, Copy)]
+pub struct RpcParams {
+    pub start_block: u64,
+    pub end_block: u64,
+    pub checkpoint_block_number: Option<u64>,
+    pub rpc_type: RpcType,
+}
```

Comment:
There is already an `RpcParams` struct in leader/src/client.rs, maybe we could unify the fields and reuse one over the place?